tests/ui/lang/core/array/oob_0_array.rs

```diff
@@ -0,0 +1,11 @@
+#![allow(unconditional_panic)]
```

Comment:
Is this needed?

Reply:
Those are needed, otherwise it hits the lint and compilation stops.

In theory, most users would stop there, since they see it would panic. But in some rare cases people may try to avoid the lint and keep compiling and we shouldn't fail with a cryptic error message.

Reply:
👍